packages/bpk-component-navigation-tab-group/src/BpkNavigationTabGroup-test.tsx

```diff
@@ -0,0 +1,106 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import BpkNavigationTabGroup from './BpkNavigationTabGroup';
+
+import type { Props } from './BpkNavigationTabGroup';
+
+const tabs: Props['tabs'] = [
+  { text: 'Flights', href: '/' },
+  { text: 'Hotels', href: '/hotel' },
+  { text: 'Car hire', href: '/carhire' },
+];
+
+describe('BpkNavigationTabGroup', () => {
+  beforeEach(() => {
+    window.matchMedia = jest.fn().mockImplementation(() => ({
+      matches: true,
+      addEventListener: jest.fn(),
+      removeEventListener: jest.fn(),
+    }));
+  });
+
+  it('should render correctly', () => {
+    const { asFragment } = render(
+      <BpkNavigationTabGroup
+        tabs={tabs}
+        onItemClick={() => {}}
+        selectedIndex={0}
+      />,
+    );
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  it('should render selected link', () => {
+    render(
+      <BpkNavigationTabGroup
+        tabs={tabs}
+        onItemClick={() => {}}
+        selectedIndex={0}
+      />,
+    );
+
+    const selectedTextElement = screen.getByText('Flights');
+    const selectedLink = selectedTextElement.closest('a');
+
+    expect(selectedLink).toHaveClass(
+      'bpk-navigation-tab-wrap--surface-contrast-selected',
+    );
+  });
+
+  it('should call onItemClick when a tab is clicked', async () => {
+    const user = userEvent.setup();
+
+    const onItemClick = jest.fn();
+
+    render(
+      <BpkNavigationTabGroup
+        tabs={tabs}
+        onItemClick={onItemClick}
+        selectedIndex={0}
+      />,
+    );
+
+    await user.click(screen.getByText('Hotels'));
+
+    expect(onItemClick).toHaveBeenCalledTimes(1);
+    expect(onItemClick).toHaveBeenCalledWith(
+      { text: 'Hotels', href: '/hotel' },
+      1,
+    );
+  });
+
+  it('should render correctly when type is CanvasDefault', () => {
+    render(
+      <BpkNavigationTabGroup
+        tabs={tabs}
+        onItemClick={() => {}}
+        selectedIndex={0}
+        type="CanvasDefault"
+      />,
+    );
+
+    const flightTextElement = screen.getByText('Flights');
+    const flightLink = flightTextElement.closest('a');
+
+    expect(flightLink).toHaveClass('bpk-navigation-tab-wrap--CanvasDefault');
```

Comment:
Shouldn't this be 

```suggestion
    expect(flightLink).toHaveClass('bpk-navigation-tab-wrap--canvas-default');
```

Reply:
Updated in[ commit1](https://github.com/Skyscanner/backpack/pull/3599/commits/911824c84a19d66760a34331e8e044eb2efc9b05#diff-be40bf36991c4636669cf858f102389270bca168ce01f6f5ec479bac465bdca3R104) and [commit2](https://github.com/Skyscanner/backpack/pull/3599/commits/911824c84a19d66760a34331e8e044eb2efc9b05#diff-be40bf36991c4636669cf858f102389270bca168ce01f6f5ec479bac465bdca3R104)